chromium_src/chrome/browser/sessions/session_common_utils.cc

```diff
@@ -25,5 +25,7 @@ bool ShouldTrackURLForRestore(const GURL& url) {
     return false;
   }
 #endif
+  if (url.spec() == kBraveOnboardingRestoreWalletURL)
```

Comment:
(Sorry if it's too early for feedback, but I just noticed this line.)

Please compare each individual component of the URL separately (e.g. scheme, hostname) instead of using a string comparison. String comparisons of URLs often lead to security vulnerabilities.

Reply:
updated, thanks